sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/EncryptedBlobClientBuilder.java

```diff
@@ -915,4 +919,17 @@ public EncryptedBlobClientBuilder requiresEncryption(boolean requiresEncryption)
         this.requiresEncryption = requiresEncryption;
         return this;
     }
+
+    /**
+     * Sets the partition chunk size
+     * @param gcmEncryptionRegionLength
+     * @return
+     */
+    public EncryptedBlobClientBuilder gcmEncryptionRegionLength(int gcmEncryptionRegionLength) {
```

Comment:
Perhaps we call this `authenticatedRegionDataLength`. Data sizes like this aren't specific to GCM, so we don't have to lock this name into that specific encryption mode.